gradle.properties

```diff
@@ -23,10 +23,11 @@ kotlin_version=1.6.0
 kotlin.native.ignoreDisabledTargets=true
 kotlin.mpp.stability.nowarn=true
 kotlin.mpp.enableCompatibilityMetadataVariant=true
+kotlin.native.binary.memoryModel=experimental
```

Comment:
What happens, if the consumer of Ktor does not use the experimental MM? Do you need a check, like a `native-mt` one too?

Reply:
Nope, we drop native-mt support

Reply:
I know, I mean a check: `isExperimentalMMActive`

Reply:
I'm not sure that if will be possible, it's likely to fail with `InvalidImmutablityException`